Update .travis.yml

JGarcia-Panach opened the pull request:
Travis configuration for transifex.
Following the template in https://github.com/OCA/maintainer-quality-tools/blob/master/sample_files/.travis.yml
and catching the secure from OCA/maintainer-quality-tools#194
Thanks @pedrobaeza

JGarcia-Panach committed:
Update .travis.yml

Travis configuration for transifex.
Following the template in https://github.com/OCA/maintainer-quality-tools/blob/master/sample_files/.travis.yml
and cathing the secure from OCA/maintainer-quality-tools#194
Thanks @pedrobaeza

pedrobaeza commented on .travis.yml:
```diff
@@ -17,6 +17,36 @@ env:
   - VERSION="8.0" UNIT_TEST="1" LINT_CHECK="0"
+  - VERSION="8.0" TESTS="0" LINT_CHECK="0" TRANSIFEX="0"
+  - TRANSIFEX_USER='transbot@odoo-community.org'
+  # This line contains the encrypted transifex password
```
You don't need to copy all comment lines. They are there for indicating what to do, but not to be on each final file.

pedrobaeza commented on .travis.yml:
```diff
@@ -17,6 +17,36 @@ env:
   - VERSION="8.0" ODOO_REPO="OCA/OCB" LINT_CHECK="0"
   - VERSION="8.0" UNIT_TEST="1" LINT_CHECK="0"
+  - VERSION="8.0" TESTS="0" LINT_CHECK="0" TRANSIFEX="0"
+  - TRANSIFEX_USER='transbot@odoo-community.org'
```
This is not located in the right part

pedrobaeza commented:
Have you made the commit from other account?

Another thing: please take the occasion to change the coveralls call by the new one (travis_after_success or similar)

pedrobaeza commented:
Thank you for making this!

JGarcia-Panach commented:
Thanks to you Pedro, first thing in the morning I will check all your comments and try to fix them

JGarcia-Panach committed:
Update .travis.yml

Deleting comment lines
global was missing in previous update file
changing coveralls by travis_tests_after_success

JGarcia-Panach commented:
With your comments @pedrobaeza and checking other OCA .travis files, I think it should be correct now.
I have made the commit with this account (the only I have in github)
Thanks again for all your help

pedrobaeza commented:
Yeah, it's correct! :+1: 

I'm saying about the GitHub account because of the CLA. Did you sign it, right?

JGarcia-Panach commented:
I've signed it this morning.
https://github.com/odoo/odoo/pull/10958

Do I also have to sign ICLA.pdf and send to cla AT odoo-community.org? or it is enough with the PR...

pedrobaeza commented:
The OCA CLA is not the same. You can see here ours: https://odoo-community.org/page/cla

JGarcia-Panach commented:
I'm doing it right now. Sorry for all the trouble.

pedrobaeza commented:
No problem. Thankful for seeing you contributing.

legalsylvain commented:
Hi @JGarcia-Panach,
did you finally send it ? I'd like to merge this PR and I don't see your CLA in OCA database.

kind regards.

JGarcia-Panach commented:
Yes I sent it, @legalsylvain

If there is any problem let me know.

Thanks!!

atchuthan commented:
:+1:
